### PR TITLE
Pl 802xx relnotes

### PR DESCRIPTION
**Pull request by deepakbolangady**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore
- [x] Release notes

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [x] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -54,6 +51,61 @@ This release does not include any early access features.
...
+
+- The Setup Vault step of the Azure Key Vault connector wizard continued to use the previous secret reference even after you changed it. (PL-40269, ZD-48150)
+
+  The issue is now fixed, and the wizard reads the latest value. 
+
+- Harness Manager did not show role assignments for a service account at the organization and project scopes if the parent scope (the account scope and organization scope, respectively) had a service account with the same identifier. (PL-40245, ZD-47906)
+
+  The issue is now fixed, and any affected service accounts will be displayed correctly. 
+
+- Harness API requests for creating an API key for a service account returned an HTTP 200 OK status code even if the API request did not have the valid service account ID. Consequently, the harness_platform_apikey resource for Terraform failed to create the API key at the intended scope. The issue was caused by Harness failing to validate the service account identifier. (PL-40243, ZD-47921)
```
Comment (brian-f-harness) [inline release-notes/platform.md:72]:
```suggestion
- Harness API requests for creating an API key for a service account returned an HTTP 200 OK status code even if the API request did not have a valid service account ID. Consequently, the `harness_platform_apikey` resource for Terraform failed to create the API key at the intended scope. The issue was caused by Harness failing to validate the service account identifier. (PL-40243, ZD-47921)
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -54,6 +51,61 @@ This release does not include any early access features.
...
+## Previous releases
+
+<details>
+<summary>2023 releases</summary>
+
+#### August 4, 2023, version 80120
+
+##### What's new
+
+- The Universal Base Image Minimal used for the Harness user interface (both FirstGen and NextGen) and the Harness NextGen authentication service has been upgraded to ubi8-minimal:8.8. This upgrade was necessitated by version 8.7 (ubi8-minimal:8.7) reaching end of life. (PL-40095)
```
Comment (brian-f-harness) [inline release-notes/platform.md:98]: Announced last release:
https://developer.harness.io/release-notes/whats-new/#august-4-2023
Do we need to include again?

**Review by brian-f-harness — APPROVED** [review]
LGTM. Left a couple suggestions, added the delegate component ticket, and had a question about a previous release note we've included here.

**Review by StrawberryPad — APPROVED** [review]

File: release-notes/platform.md (hunk 2 above)
Comment (deepakbolangady) [reply release-notes/platform.md:98]: Hi, @brian-f-harness. Which last release? I checked releases prior to Aug 4. Didn't find it anywhere.

**Review by charanya-j — APPROVED** [review]

**Pull request merged**